agent/agent/src/main/java/com/microsoft/applicationinsights/agent/Agent.java

```diff
@@ -42,6 +42,8 @@ public static void premain(String agentArgs, Instrumentation inst) {
     }
     OpenTelemetryAgent.premain(agentArgs, inst);
     alreadyLoaded = true;
+
+    StartupProfiler.start();
```

Comment:
I'm not sure, but suspect this may fix the ClassLoader issue by moving this to the beginning (which we want anyways)

```suggestion
    StartupProfiler.start();

    OpenTelemetryAgent.premain(agentArgs, inst);
    alreadyLoaded = true;
```